.gitattributes

```diff
@@ -1 +1,2 @@
-**/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
\ No newline at end of file
+**/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
+baseline-prof.txt linguist-generated=true
```

Comment:
I think we maybe need to do a glob like `**/baseline-prof.txt`

Reply:
hmm.. I don't think that was it. I'm still seeing the `baseline-prof.txt` show up.

Reply:
hmm, ok let's figure that out as a followup